### PR TITLE
refactor(tests): Refactor tests, remove unnecessary code

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
This PR refactors and simplify tests as requested in #21 

It is still not optimal but I would not spend more time on this because this repo is going to be sooner or later rewritten to monorepo anyway.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    "es2015",
+    "stage-0"
+  ],
+  "plugins": [
+    "transform-async-to-generator",
+    "istanbul",
+    "transform-async-to-generator"
```
Comment (wizzy25) [inline .babelrc]: I think this should go. Double declaration

File: tests/integration/product-type-export-encoding.spec.js [hunk 2]
```diff
@@ -112,45 +79,46 @@ test(`productType export module
...
-          expectedFileName1,
-          `file name is ${expectedFileName1}`,
-        )
-        t.equal(
-          files[1].split('/').pop(),
-          expectedFileName2,
-          `file name is ${expectedFileName2}`,
-        )
-        resolve()
+  return before()
```
Comment (wizzy25) [inline tests/integration/product-type-export-encoding.spec.js]: +1 For going the async way above. We can also await on before, to make the code even more compact

File: tests/integration/product-type-export-xlsx.spec.js [hunk 3]
```diff
@@ -214,279 +41,273 @@ const readXlsx = function readXlsx (filePath) {
...
-      )
-      .then((data) => {
-        const attributes = productTypeExport.attributeNames.join(',')
-        const header = data[0].join(',')
-        t.equal(
-          header,
-          `name,key,description,${attributes}`,
-          `exported row ${header} is equal`
+    t.timeoutAfter(15000) // 15s
+    before().then(() => {
```
Comment (wizzy25) [inline tests/integration/product-type-export-xlsx.spec.js]: Here too. We can pass an anonymous `async` function to test and declare `await before()`

File: tests/integration/product-type-export-xlsx.spec.js [hunk 4]
```diff
@@ -214,279 +41,273 @@ const readXlsx = function readXlsx (filePath) {
...
-          {
-            productTypes: testProductTypes.length,
-            attributes: mockAttributes.length,
-          },
-          'Summary report is valid',
-        )
-        t.end()
-      })
+    t.timeoutAfter(15000) // 15s
+    before().then(() => {
```
Comment (wizzy25) [inline tests/integration/product-type-export-xlsx.spec.js]: Here too

File: tests/integration/product-type-export-xlsx.spec.js [hunk 5]
```diff
@@ -214,279 +41,273 @@ const readXlsx = function readXlsx (filePath) {
...
-            `file name is ${expectedFileName1}`,
-          )
-          t.equal(
-            path.parse(files[1]).base,
-            expectedFileName2,
-            `file name is ${expectedFileName2}`,
-          )
-          t.end()
+    t.timeoutAfter(15000) // 15s
+    before().then(() => {
```
Comment (wizzy25) [inline tests/integration/product-type-export-xlsx.spec.js]: ^^^

File: tests/utils.js [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+import { randomBytes } from 'crypto'
+import { SphereClient } from 'sphere-node-sdk'
+
+import getSphereClientCredentials from '../src/sphere-client-credentials'
+
+export const PROJECT_KEY = process.env.CI === 'true'
+  ? process.env.SPHERE_PROJECT_KEY
+  : process.env.npm_config_projectkey
+
+export async function getClientConfig (projectKey = PROJECT_KEY) {
```
Comment (wizzy25) [inline tests/utils.js]: You don't need to declare this as an `async` function

File: .babelrc (hunk 1 above)
Comment (junajan) [reply .babelrc]: Hello @wizzy25 .. updated here https://github.com/sphereio/sphere-product-type-export/pull/22/commits/0a632e9f8e605d0a558865faec9e5308a357eb5e#diff-e56633f72ecc521128b3db6586074d2cL9

File: tests/integration/product-type-export-encoding.spec.js (hunk 2 above)
Comment (junajan) [reply tests/integration/product-type-export-encoding.spec.js]: Most of tests updated here 0a632e9

File: tests/integration/product-type-export-xlsx.spec.js (hunk 3 above)
Comment (junajan) [reply tests/integration/product-type-export-xlsx.spec.js]: Updated here 0a632e9

File: tests/integration/product-type-export-xlsx.spec.js (hunk 5 above)
Comment (junajan) [reply tests/integration/product-type-export-xlsx.spec.js]: Updated here 0a632e9

File: tests/integration/product-type-export-xlsx.spec.js (hunk 4 above)
Comment (junajan) [reply tests/integration/product-type-export-xlsx.spec.js]: Updated here 0a632e9

File: tests/utils.js (hunk 6 above)
Comment (junajan) [reply tests/utils.js]: Async removed here https://github.com/sphereio/sphere-product-type-export/pull/22/commits/0a632e9f8e605d0a558865faec9e5308a357eb5e#diff-74a635be9be46d0f8b20784f5117bb0cR11

**Pull request merged**